daffodil-test/src/test/resources/org/apache/daffodil/section07/escapeScheme/escapeScenarios.tdml

```diff
@@ -551,74 +409,6 @@
     </errors>
   </parserTestCase>
 
-  <parserTestCase name="scenario2_12" model="es2"
-    description="Section 13 - escapeCharacter - DFDL-13-029R" root="e_infix_never"
-    roundTrip="onePass">
-    <document>foo$/$;</document>
-    <infoset>
-      <dfdlInfoset>
-        <tns:e_infix_never>
-          <x>foo/</x>
-          <y></y>
-        </tns:e_infix_never>
-      </dfdlInfoset>
-    </infoset>
-  </parserTestCase>
-
-  <parserTestCase name="scenario2_12_postfix" model="es2"
-    description="Section 13 - escapeCharacter - DFDL-13-029R" root="e_postfix">
-    <document>foo$/$;</document>
-    <infoset>
-      <dfdlInfoset>
-        <tns:e_postfix>
-          <x>foo/</x>
-        </tns:e_postfix>
-      </dfdlInfoset>
-    </infoset>
-  </parserTestCase>
-
-  <parserTestCase name="scenario2_13" model="es2"
-    description="Section 13 - escapeCharacter - DFDL-13-029R" root="e_infix_keepEmpty" roundTrip="twoPass">
-    <!-- This is twoPass as there is no element 'y' in the infoset, so on unparse there is no separator -->
-    <document>foo$$/$;</document>
-    <infoset>
-      <dfdlInfoset>
-        <tns:e_infix_keepEmpty>
-          <x>foo$/</x>
-          <!-- dfdlx:emptyElementParsePolicy 'treatAsEmpty' only creates empty
-               string elements if there is some non-zero-length syntax
-          <y></y>
-          -->
-        </tns:e_infix_keepEmpty>
-      </dfdlInfoset>
-    </infoset>
-  </parserTestCase>
-
-  <parserTestCase name="scenario2_13_postfix" model="es2"
-    description="Section 13 - escapeCharacter - DFDL-13-029R" root="e_postfix" roundTrip="true">
-    <document>foo$$/$;</document>
-    <infoset>
-      <dfdlInfoset>
-        <tns:e_postfix>
-          <x>foo$/</x>
-        </tns:e_postfix>
-      </dfdlInfoset>
-    </infoset>
-  </parserTestCase>
-
-  <parserTestCase name="scenario2_14" model="es2"
-    description="Section 13 - escapeCharacter - DFDL-13-029R" root="e_infix" roundTrip="false">
-    <!-- See DFDL-DAFFODIL for to make roundTrip="true" -->
-    <document>foo$$$//$;</document>
-    <infoset>
-      <dfdlInfoset>
-        <tns:e_infix>
-          <x>foo$$/$;</x>
-  </tns:e_infix>
-      </dfdlInfoset>
-    </infoset>
-  </parserTestCase>
-
```

Comment:
The comment line just below should get a bug report. A bad diagnostic like that is a bug.

Reply:
Just double checked this one and the only message that gets displayed now is "Parse Error: Terminator '$;' not found" so something must have changed since it was originally written.  I'll remove the comment.